src/hssm/param/utils.py

```diff
@@ -0,0 +1,43 @@
+"""Utility functions for the parameter classes."""
+
+import bambi as bmb
+import numpy as np
+
+
+def validate_bounds(bounds: tuple[float, float]) -> None:
+    """Validate the bounds."""
```

Comment:
I think the docstring could be improved by adding some more context or maybe an example.
```suggestion
    """
    Validate the given bounds.

    This function checks that the `bounds` are a valid tuple containing two values,
    where the first value is less than the second also handling unbounded cases.

        Parameters
        ----------
        bounds (tuple[float, float]): A tuple containing two float values representing 
                                       the lower and upper bounds.

        Raises
        -------
        ValueError: If `bounds` does not contain exactly two values, or if the 
                    lower bound is greater than or equal to the upper bound.

        Examples:
        >>> validate_bounds((0.0, 1.0))  # Valid input, should not raise an exception

        >>> validate_bounds((1.0, 0.0))  # Invalid input, should raise ValueError
        Traceback (most recent call last):
            ...
        ValueError: Invalid bounds: (1.0, 0.0)

        >>> validate_bounds((0.0, 0.0))  # Invalid input, should raise ValueError
        Traceback (most recent call last):
            ...
        ValueError: Invalid bounds: (0.0, 0.0)

        >>> validate_bounds((-inf, inf))  # Valid input, should not raise an exception

        >>> validate_bounds((inf, -inf))  # Invalid input, should raise ValueError
        Traceback (most recent call last):
            ...
        ValueError: Invalid bounds: (inf, -inf)
    """
```

Reply:
Sounds great!